### PR TITLE
Update documentation for connection string parameters

### PR DESCRIPTION
**Pull request by vonzshik**

### CODE REVIEW

File: conceptual/Npgsql/connection-string-parameters.md [hunk 1]
```diff
@@ -78,6 +79,13 @@ Target Session Attributes | Determines the preferred PostgreSQL target server ty
 Load Balance Hosts        | Enables balancing between multiple hosts by round-robin.                   | false
 Host Recheck Seconds      | Controls for how long the host's cached state will be considered as valid. | 10
 
+## Multiplexing
+
+Parameter                                      | Description                                                                                                  | Default
+---------------------------------------------- | -----------------------------------------------------------------------------------------------------------  | -------------------------
+Multiplexing                                   | Enables multiplexing, which allows more efficient use of connections. Introduced in 5.0.                     | false
```
Comment (roji) [inline conceptual/Npgsql/connection-string-parameters.md:86]: I'm still hesitant to introduce this without a serious documentation page which explains exactly how to use it. I'm worried users would turn it on and use it incorrectly. For example, in multiplexing it's very important to consume the resultset ASAP so as to release the connection to other consumers, otherwise you're blocking the entire application; to make this really usable, I think we should do resultset buffering by default when using multiplexing.

In other words, is it OK if we defer documenting these parameters until more proper general documentation is available?

File: conceptual/Npgsql/connection-string-parameters.md [hunk 2]
```diff
@@ -87,9 +95,12 @@ Application Name         | The optional application name parameter to be sent to
...
+Encoding                 | Gets or sets the .NET encoding that will be used to encode/decode PostgreSQL string data.            | UTF8
 Timezone                 | Gets or sets the session timezone.                                                                   | PGTZ
 EF Template Database     | The database template to specify when creating a database in Entity Framework.                       | template1
+EF Admin Database        | The database admin to specify when creating and dropping a database in Entity Framework.             | template1
 Load Table Composites    | Load table composite type definitions, and not just free-standing composite types.                   | false
+ArrayNullabilityMode     | Configure the way arrays of value types are returned when requested as object instances. Possible values are: Never (arrays of value types are always returned as non-nullable arrays), Always (arrays of value types are always returned as nullable arrays) and PerInstance (the type of array that gets returned is determined at runtime).             | Never
```
Comment (roji) [inline conceptual/Npgsql/connection-string-parameters.md:103]: This is a mistake - it should be `Array Nullability Mode` (with spaces), we should correct this in the code...

**Review by roji — APPROVED** [review]

File: conceptual/Npgsql/connection-string-parameters.md (hunk 1 above)
Comment (vonzshik) [reply conceptual/Npgsql/connection-string-parameters.md:86]: Sure, I'll remove this part.
